src/components/PlanterQuote.js

```diff
@@ -0,0 +1,98 @@
+import { Box, Stack, Typography } from '@mui/material';
+import { makeStyles } from 'models/makeStyles';
+import Image from 'next/image';
+import React from 'react';
+
+import DataTag from './common/DataTag';
+
+const quoteImg = '/images/quote-symbol.svg';
+
+const useStyles = makeStyles()((theme) => ({
+  container: {
+    background: theme.palette.background.OrangeGreenGradientDark,
+    display: 'flex',
+    boxSizing: 'border-box',
+    height: 'fit-content',
+    padding: theme.spacing(6),
+    paddingTop: theme.spacing(15),
+    paddingBottom: theme.spacing(15),
+    [theme.breakpoints.down('sm')]: {
+      padding: theme.spacing(13),
+      paddingTop: theme.spacing(15),
+      paddingBottom: theme.spacing(8),
+      marginTop: theme.spacing(30),
```

Comment:
Padding should to be used for arranging elements like this. The root element should also not have any margins as that effects reuseability. `alignItems: 'center'` and `justifyContent: 'center'` should be adequate for this.